### PR TITLE
Update XPU install

### PR DESCRIPTION
**Pull request by kprokofi**
### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have ran e2e tests and there is no issues.
- [ ] I have added the description of my changes into CHANGELOG in my target branch (e.g., [CHANGELOG](https://github.com/openvinotoolkit/training_extensions/blob/develop/CHANGELOG.md) in develop).​
- [ ] I have updated the documentation in my target branch accordingly (e.g., [documentation](https://github.com/openvinotoolkit/training_extensions/tree/develop/docs) in develop).
- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

### License

- [ ] I submit _my code changes_ under the same [Apache License](https://github.com/openvinotoolkit/training_extensions/blob/develop/LICENSE) that covers the project.
      Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2024 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -67,6 +67,22 @@ docs = [
     "sphinxemoji",
     "nbsphinx",
 ]
+xpu = [
+    "torch==2.1.0.post0",
+    "torchvision==0.16.0.post0",
+    "intel-extension-for-pytorch==2.1.20+xpu",
```
Comment (sovrasov) [inline pyproject.toml]: Please double-check https://intel.github.io/intel-extension-for-pytorch/index.html#installation?platform=gpu&version=v2.1.30%2bxpu&os=linux%2fwsl2&package=pip

File: docs/source/guide/get_started/installation.rst [hunk 2]
```diff
@@ -76,49 +76,35 @@ OpenVINO™ Training Extensions command line functionality.
...
+.. tab-set::
 
-* A local source in development mode
+    .. tab-item:: Minimum requirements
 
-.. code-block:: shell
+        .. code-block:: shell
 
-    pip install -e .
+            pip install '.[xpu_full]' --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/xpu/us/
```
Comment (eunwoosh) [inline docs/source/guide/get_started/installation.rst]:
```suggestion
            pip install '.[xpu]' --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/xpu/us/
```
It seems typo

File: docs/source/guide/get_started/installation.rst [hunk 3]
```diff
@@ -76,49 +76,35 @@ OpenVINO™ Training Extensions command line functionality.
...
 
-* PyPI
+        .. note::
 
-.. code-block:: shell
+            Models from mmlab are not available for this environment. If you want to use mmlab models, you must install them with Full Requirements.
+            Also, some tasks may not be supported by minimum requirements.
 
-    pip install otx
+    .. tab-item:: Full Requirements
```
Comment (eunwoosh) [inline docs/source/guide/get_started/installation.rst:124]: torch installation step is necessary for 'Full requirement'.

**Review by eunwoosh — COMMENTED** [review]
Thanks for your work. I leave some comments. Please take a look.

**Review by harimkang — APPROVED** [review]

**Review by sovrasov — APPROVED** [review]

**Pull request merged**